packages/document/main-doc/docs/en/configure/app/tools/postcss.mdx

```diff
@@ -9,27 +9,31 @@ configName: tools.postcss
 - **Default:**
 
 ```js
-const defaultOptions = {
-  postcssOptions: {
-    plugins: [
-      // 以下插件默认启用
-      require('postcss-nesting'),
-      require('postcss-media-minmax'),
-      require('postcss-flexbugs-fixes'),
-      require('autoprefixer')({
-        flexbox: 'no-2009',
-      }),
-      // 以下插件仅在需要兼容低版本浏览器时启用
-      require('postcss-custom-properties'),
-      require('postcss-initial'),
-      require('postcss-page-break'),
-      require('postcss-font-variant'),
-    ],
-    // 默认在开发环境下启用 CSS 的 Source Map
-    sourceMap: isDev,
-  },
-};
+const defaultOptions =
+  tools.lightningcssLoader === false
```

Comment:
Can we add a tip to this documentation instead of update the `defaultOptions`?

The `defaultOptions` example should not become too complex.